asciidoc/etl-tool.adoc

```diff
@@ -2,6 +2,10 @@
 :docs: 
 :slug: etl-tool
 :imagesdir: https://s3.amazonaws.com/dev.assets.neo4j.com/wp-content/uploads
+:author: Michael Hunger, Larus BA
```

Comment:
I'm not 100% sure but I think the WordPress plugin will take the first author, so the author on this page will be "Michael Hunger" on this page (sorry Larus BA :wink:)